terraform/aws/public-cloud/main.tf

```diff
@@ -1,8 +1,19 @@
 variable "access_key" {}
 variable "secret_key" {}
 variable "organization" { default = "kubeform" }
-variable "region" { default = "eu-west-1" }
-variable "availability_zones" { default = "eu-west-1a,eu-west-1b,eu-west-1c" }
+#variable "region" { default = "eu-west-1" }
+#variable "availability_zones" { default = "eu-west-1a,eu-west-1b,eu-west-1c" }
```

Comment:
Why dont we keep "eu-west-1a,eu-west-1b,eu-west-1c" for the new definition? why only 2 elements?